### PR TITLE
Updates issue templates

### PR DESCRIPTION
**Pull request by PowerSchill**
Reformatted issue template and added feature request template to point to new feature repository.

Signed-off-by: Mark E. Schill <Mark.Schill@cmschill.net>

Closes #
Closes #
Closes #

# Changes proposed in this pull request:
-
-
-

Review please

### CODE REVIEW

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,47 @@
-# Before reporting please make sure that this issue was not reported yet (Check out similar issue suggestions).
-## Also update to the latest alpha of both minecolonies and structurize before reporting
+# Prerequisites
 
-<!-- If you are making a bug report, please fill out this template as best as possible, this will help 
+- [ ] I am running the lastest version of Minecolonies for my minecraft version.
```
Comment (ravenbuilder934) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- [ ] I am running the latest version of MineColonies for my Minecraft version.
```

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
+- [ ] I checked the Minecolonies [Wiki](https://ldtteam.github.io/MinecoloniesWiki/) and made sure that my issue is not covered by any content.
```
Comment (ravenbuilder934) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- [ ] I checked the MineColonies [Wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
```

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
+- [ ] I checked and made sure that **this issue is not already covered by any existing issue**.
```
Comment (ravenbuilder934) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- [ ] I made sure that **this issue is not a duplicate of any existing issue.**
```

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -1,38 +1,47 @@
...
+### Context
+
 <!-- Exact version, eg: 0.9.126-ALPHA or 0.9.2-RELEASE, please don't just give the Minecraft version you're playing.
 Before reporting: Update to the newest minecolonies alpha and see if your issue might be fixed already.
 -->
 
-Version: 
+- Minecraft Version:
+- Minecolonies Version:
+- Related mods & Versions:
```
Comment (ravenbuilder934) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- Related Mods and Their Versions:
```

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -1,38 +1,47 @@
...
 
+### Context
+
 <!-- Exact version, eg: 0.9.126-ALPHA or 0.9.2-RELEASE, please don't just give the Minecraft version you're playing.
 Before reporting: Update to the newest minecolonies alpha and see if your issue might be fixed already.
 -->
 
-Version: 
+- Minecraft Version:
+- Minecolonies Version:
```
Comment (ravenbuilder934) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- MineColonies Version:
```

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -1,38 +1,47 @@
-# Before reporting please make sure that this issue was not reported yet (Check out similar issue suggestions).
-## Also update to the latest alpha of both minecolonies and structurize before reporting
+# Prerequisites
 
-<!-- If you are making a bug report, please fill out this template as best as possible, this will help 
+- [ ] I am running the lastest version of Minecolonies for my minecraft version.
+- [ ] I checked the Minecolonies [Wiki](https://ldtteam.github.io/MinecoloniesWiki/) and made sure that my issue is not covered by any content.
+- [ ] I checked and made sure that **this issue is not already covered by any existing issue**.
+- [ ] This is an issue and not a feature request. Please create all feature requests at [Minecolonies-features](https://github.com/ldtteam/minecolonies-features/issues).
```
Comment (ravenbuilder934) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- [ ] This is an issue and not a feature request. Please create all feature requests [here](https://github.com/ldtteam/minecolonies-features/issues).
```

Comment (ravenbuilder934) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: Is someone going to copy everything from #4535 to this new repo?

File: .github/workflows/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Notice
+
+Feature requests for Minecolonies are located in the [minecolonies-feature](https://github.com/ldtteam/minecolonies-features/issues) repository. Please cancel this request and generate the request in that repository.
```
Comment (ravenbuilder934) [inline .github/workflows/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
Feature requests for MineColonies are located [here](https://github.com/ldtteam/minecolonies-features/issues). Please cancel this issue creation and create an issue in that repository.
```

**Review by ravenbuilder934 — CHANGES_REQUESTED** [review]

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -1,38 +1,47 @@
-# Before reporting please make sure that this issue was not reported yet (Check out similar issue suggestions).
-## Also update to the latest alpha of both minecolonies and structurize before reporting
+# Prerequisites
 
-<!-- If you are making a bug report, please fill out this template as best as possible, this will help 
+- [ ] I am running the latest version of MineColonies for my Minecraft version.
```
Comment (someaddons) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md]: add alpha here, many see release as latest

**Pull request merged**